### PR TITLE
Map through team objects

### PR DESCRIPTION
**Pull request by RickyWang1234**
Screenshot 2023-10-30 133322

Screenshot 2023-10-30 132912

### CODE REVIEW

**Comment by RickyWang1234** [comment]
i had a fill in for the title, I am unsure if I am suppose to create the pictures myself or if there is one provided in the files

File: src/app/layout.js [hunk 1]
```diff
@@ -23,6 +24,7 @@ export default function RootLayout({ children }) {
         <Navigation />
         {children}
         <Footer />
+        <Teammate />
```
Comment (pondohoo) [inline src/app/layout.js:27]: move this to team/page.js

File: src/components/team/Teammate.jsx [hunk 2]
```diff
@@ -1,5 +1,97 @@
...
+import Mordekaiser from "public/team/mordekaiser.png";
+import Gragas from "public/team/gragas.png";
+import Aphelios from "public/team/aphelios.png";
+import Rell from "public/team/rell.png";
+
 const Teammate = () => {
-  return <div>Teammate</div>;
+  return (
+    <div>
+      <div className="flex justify-center font-bold text-4xl shadow-md pb-3">
```
Comment (pondohoo) [inline src/components/team/Teammate.jsx:12]: Use the \<Heading\> component for this

**Review by pondohoo — COMMENTED** [review]
now that all the members are laid out can you make it look like this
image

image

**Review by pondohoo — APPROVED** [review]

**Pull request merged**